CHROM: chromosome number
POS: genomic position
ID: ClinVar identifier
REF: reference allele
ALT: alternate allele
CLNDN: clinical diagnosis
CLNREVSTAT: status of evidence
CLNSIG: ClinVar interpretation
CLNSIGCONF: confidence of interpretation
RS: rs number from dbSNP
Pathogenicity: groundtruth
Transcript: transcript for most deleterious effect
Consequence: most deleterious functional effect
Gene: gene symbol:
AAChange: amino acid change:
pLI: genetic constraint score:
MIM: phenotype from OMIM
GERP: conservation score
Eigen:	predictor for variant pathogenicity
FATHMM: predictor for variant pathogenicity
M_CAP: predictor for variant pathogenicity
MetaRNN: predictor for variant pathogenicity
MetaSVM: predictor for variant pathogenicity
MPC: predictor for variant pathogenicity
MutationAssessor: predictor for variant pathogenicity
MutationTaster: predictor for variant pathogenicity
MVP: predictor for variant pathogenicity
Polyphen2_HVAR: predictor for variant pathogenicity
PrimateAI: predictor for variant pathogenicity
PROVEAN: predictor for variant pathogenicity
SIFT4G: predictor for variant pathogenicity
gMVP: predictor for variant pathogenicity
VEST4: predictor for variant pathogenicity
REVEL: predictor for variant pathogenicity
CADD: predictor for variant pathogenicity
EVE: prediction label from EVE
gnomAD_AF_popmax: maximum allele frequency across ancestry from gnomAD2
gnomAD3_AF: overall allele frequency from gnomAD3
UKBB_AF: overall allele frequency from UKBB 

